pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -118,6 +118,6 @@ stdenv.mkDerivation rec {
     homepage = "http://fontconfig.org/";
     license = licenses.bsd2; # custom but very bsd-like
     platforms = platforms.all;
-    maintainers = [ maintainers.vcunat ];
+    maintainers = [ ];
```

Comment:
@jtojnar: perhaps the `freedesktop` team?

Reply:
Yeah, that is in scope.